Makefile

```diff
@@ -203,6 +203,11 @@ install:
 
 	install -Dm644 -t ${PREFIX}/share/man/man1 ./docs/*.1
 
+	mkdir -p ${PREFIX}/include/hyprland
```

Comment:
doing this essentially deprecates `make pluginenv`, no?

Reply:
Yes. We can remove it and instruct plugin devs to install Hyprland from source again, and then use `#include <hyprland/src/...>`.

Reply:
sure.